# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../libev/no_compat.diff
# Copyright (C) 2008 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

--- ./Makefile.am.orig	2008-09-15 16:37:48.000000000 +0300
+++ ./Makefile.am	2008-09-15 16:38:18.000000000 +0300
@@ -9,10 +9,10 @@
 
 man_MANS = ev.3
 
-include_HEADERS = ev.h ev++.h event.h
+include_HEADERS = ev.h ev++.h
 
 lib_LTLIBRARIES = libev.la
 
-libev_la_SOURCES = ev.c event.c
+libev_la_SOURCES = ev.c
 libev_la_LDFLAGS = -version-info $(VERSION_INFO)
 
